### PR TITLE
Feature/Models animations support

### PR DESCRIPTION
**Pull request by darek607**
- FBX (sfb) animations support for models
- fixed `show()` and `hide()` methods for model,
- fixed models clipping (affected models inside a scroll view)

test-result

### CODE REVIEW

File: android/src/main/java/com/magicleap/magicscript/ar/LoopedAnimator.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.magicleap.magicscript.ar
+
+import com.google.ar.sceneform.animation.ModelAnimator
+import com.google.ar.sceneform.rendering.ModelRenderable
+
+class LoopedAnimator : RenderableAnimator {
+
+    override fun play(modelRenderable: ModelRenderable) {
+        val animationsNumber = modelRenderable.animationDataCount
```
Comment (grzegorzdec) [inline android/src/main/java/com/magicleap/magicscript/ar/LoopedAnimator.kt]: maybe `animationsCount` or `animationsSize`

File: android/src/main/java/com/magicleap/magicscript/ar/ModelRenderableLoaderImpl.kt [hunk 2]
```diff
@@ -20,22 +20,43 @@ import android.content.Context
...
 
-    override fun loadRenderable(modelUri: Uri, resultCallback: (result: RenderableResult) -> Unit) {
-        ModelRenderable.builder()
-            .setSource(
-                context, RenderableSource.builder().setSource(
-                    context,
-                    modelUri,
-                    RenderableSource.SourceType.GLB
-                ) // GLB (binary) or GLTF (text)
+    override fun loadRenderable(
```
Comment (grzegorzdec) [inline android/src/main/java/com/magicleap/magicscript/ar/ModelRenderableLoaderImpl.kt]: this function is very long, is it possible to split it?

File: android/src/main/java/com/magicleap/magicscript/ar/ModelRenderableLoaderImpl.kt [hunk 3]
```diff
@@ -20,22 +20,43 @@ import android.content.Context
...
+        if (modelType == ModelType.GLB) {
+            builder.setSource(
+                context, RenderableSource.builder()
+                    .setSource(
+                        context,
+                        modelUri,
+                        RenderableSource.SourceType.GLB // GLB (binary) or GLTF (text)
+                    )
                     .setRecenterMode(RenderableSource.RecenterMode.CENTER)
                     .build()
```
Comment (grzegorzdec) [inline android/src/main/java/com/magicleap/magicscript/ar/ModelRenderableLoaderImpl.kt]: here is `.build` and in line 60 as well, is it correct?

File: android/src/main/java/com/magicleap/magicscript/ar/LoopedAnimator.kt (hunk 1 above)
Comment (darek607) [reply android/src/main/java/com/magicleap/magicscript/ar/LoopedAnimator.kt]: This is ARCore's property, cannot change :-)

File: android/src/main/java/com/magicleap/magicscript/ar/ModelRenderableLoaderImpl.kt (hunk 3 above)
Comment (darek607) [reply android/src/main/java/com/magicleap/magicscript/ar/ModelRenderableLoaderImpl.kt]: Yes, it seems incorrect, but the first build is related to `RenderableSource.builder()` but the second  is for `ModelRenderable.builder()`

File: android/src/main/java/com/magicleap/magicscript/ar/LoopedAnimator.kt (hunk 1 above)
Comment (grzegorzdec) [reply android/src/main/java/com/magicleap/magicscript/ar/LoopedAnimator.kt]: I mean val name

Comment (darek607) [reply android/src/main/java/com/magicleap/magicscript/ar/LoopedAnimator.kt]: Hah, I'm sleeping still, ok will rename it

**Pull request merged**